### PR TITLE
readme

### PR DESCRIPTION
**Pull request by masterujjval**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by qingyun-wu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ In addition, you can find:
 ## Contributing
 
 This project welcomes contributions and suggestions. Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+Contributor License Agreement (CLA) declaring that you have the right to, and do, grant us
```
Comment (sonichi) [inline README.md:119]: Please do not modify this section. The changes in other sections are good.

**Review by sonichi — COMMENTED** [review]
The contributor license agreement is required for merging.

**Review by gagb — APPROVED** [review]

**Comment by kevin666aa** [comment]
@masterujjval Can you merge conflicts and reverse changes in Legal Notices and Contributing subsections?

File: README.md (hunk 1 above)
Comment (masterujjval) [reply README.md:119]: Sure i will revert the changes

**Comment by masterujjval** [comment]
> The contributor license agreement is required for merging.

But I'm not a corporate employ I'm a student as of now.

**Comment by kevin666aa** [comment]
> > The contributor license agreement is required for merging.
> 
> But I'm not a corporate employ I'm a student as of now.

You can reply with the default option:
(default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.

**Comment by masterujjval** [comment]
> @masterujjval please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

**Comment by masterujjval** [comment]
@microsoft-github-policy-service agree [company="{your company}"]

**Comment by masterujjval** [comment]
> @masterujjval please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Pull request closed without merging**